docs/management/managing-saved-objects.asciidoc

```diff
@@ -1,81 +1,82 @@
-To open the page, click the object name in the list of objects. You can change 
-the object title, add a description, and modify the JSON that defines the 
-object properties.
+=== View, edit, and delete
 
-If you access an object whose index has been deleted, you can:
+* To view and edit an object in its associated application, click the object title.
 
-* Recreate the index so you can continue using the object.
-* Delete the object and recreate it using a different index.
-* Change the index name referenced in the object's `kibanaSavedObjectMeta.searchSourceJSON` to point to an existing
-index pattern. This is useful if the index you were working with has been renamed.
+* To show objects that use this object, so you know the 
+impact of deleting it, click the actions icon image:images/actions_icon.png[Actions icon] 
+and select *Relationships*.
 
-WARNING: Validation is not performed for object properties. Submitting an invalid 
-change will render the object unusable. A more failsafe approach is to use the 
-*Discover*, *Visualize*, or *Dashboard* pages to create new objects instead of 
-directly editing an existing one.
+* To delete one or more objects, select their checkboxes and click *Delete*.
 
 [float]
 [[managing-saved-objects-export-objects]]
-=== Export and import objects
+=== Import and export
 
-In some cases you might want to export and import a set of save objects.
-For example:
+Using the import and export commands, you can move objects between different 
+{kib} instances. This action is useful when you 
+have multiple environments for development and production. 
 
-* You might have multiple environments for development and production. The export 
-and import commands allow you to move objects between different {kib} instances.
-* You have a large number of objects to update and want to batch this process.  
+Import and export also work well when you have a large number 
+of objects to update and want to batch the process.  
 You can export the objects to JSON, make the JSON modification, and re-import 
 the objects.
 
 [float]
-==== Export objects
+==== Import
+
+You can import multiple objects in a single operation. Click *Import* and 
+navigate to the JSON file that 
+represents the objects to import.  By default, 
+objects already in {kib} are overwritten.
+
+[float]
+==== Export
 
-Select the checkboxes of the objects you want to export, and click *Export*.
-Or, to export objects by type:
+You have two options for exporting saved objects.
 
-. Click *Export everything*.
-. Select the object types you want to export. 
-. Click *Export All*.
+* Select the checkboxes of objects that you want to export, and then click *Export*.
+* Click *Export x objects*, and select which object types to export.
+
+This action exports a JSON with all your saved objects.  By default, 
+the export includes related objects. 
 
 WARNING: Exported dashboards do not include their associated index patterns.
```

Comment:
This warning can be removed. If the "include related objects" is selected, the dashboard will export everything down to the index pattern now.